onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -201,6 +201,43 @@ def duplicate_opinfo(opinfos: list[opinfo_core.OpInfo], name: str, new_names: tu
     opinfos.extend(duplicated)
 
 
+def duplicate_opinfo_for_prims(
+    opinfos: list[opinfo_core.OpInfo], name: str, prims_name: str | None = None
+):
+    """Duplicate an opinfo in the opinfo database for a prims op.
+
+    The function sets the new OpInfo to use the variation torch.ops.prims.
+    The new OpInfo will have the name "prims_{prims_name}" where `prims_name` is the
+    name of the prims op. If `prims_name` is None, it will be set to "prims_{name}".
+
+    Args:
+        opinfos: The list of opinfo_core.OpInfo to add the new opinfo to.
+        name: The name of the opinfo to duplicate.
+        prims_name: The name of the prims op. If None, it will be set to `name`.
+    """
+    if prims_name is None:
+        prims_name = name
+    # The name of the new OpInfo
+    new_name = f"prims_{prims_name}"
+    assert prims_name.startswith("prims_")
```

Comment:
Just wondering if we have such limitation, probably we should not allow uses passing prims_name into this function. We just need the name and then add the prefix directly to avoid possible confusion to caller. IMO, The function name has a postfix "_prims" which is clear enough.

Reply:
Sorry that was a mistake

Reply:
Removed